core/trino-main/src/test/java/io/trino/execution/scheduler/TestEventDrivenTaskSource.java

```diff
@@ -272,116 +269,19 @@ public void stressTest()
         testStageTaskSourceSuccess(sourceHandles, remoteSources, splits);
     }
 
-    @Test(invocationCount = INVOCATION_COUNT)
-    public void testFailures()
```

Comment:
how is that tested now?

Reply:
The failure handling logic is now trivial (there's no custom error handling code, the assumption is that Future#transform handles failures as expected). Thought that might not be worth it maintaining this extra complexity.